tests/unit/test_sqliterepo_election.c

```diff
@@ -828,9 +827,15 @@ static void test_STEP_LISTING(void) {
 	RESET();
 	i64 = m->local_id; /* -> master */
 	transition(m, EVT_LIST_OK, &i64);
-	g_assert_cmpint(m->local_id, ==, m->master_id);
-	_member_assert_CHECKING_SLAVES(m);
-	_pending(0);
+	if (!OIO_SQLITEREPO_ELECTION_ALLOW_MASTER) {
```

Comment:
It is here necessary to use the variable (`sqliterepo_allow_master`) instead of its default value (`OIO_SQLITEREPO_ELECTION_ALLOW_MASTER`).